### PR TITLE
STR-88--BE: Upload "good" research json to the participant_id table

### PR DESCRIPTION
**Pull request by mbendandour**
# Description
Adds participants labelled 'good' to the db  
# Testing
**Using Postman:**
slack me for the information that you will add to the body! Sensitive information
-Under the Header tab, put Content-Type for **Key** and application/json; charset=UTF-8 for **value**
Make sure you are using a Post request and calling route '/sendGoodParticipants'.
Hit send. 
Check your db. Creds should be in the participants table
# Installation
'Are there any dependencies the reviewer needs to download?'

### CODE REVIEW

**Review by frlzjosh — APPROVED** [review]
good!

File: app/Models/User.php [hunk 1]
```diff
@@ -32,8 +32,8 @@ class User extends MetaUser
         'password', 'remember_token',
     ];
 
-    public function research(){
-        return $this->hasOne('App\Models\Research', 'user_id');
+    public function Participant(){
```
Comment (luisjg) [inline app/Models/User.php]: This does not follow `PSR-2` standard

File: app/Services/AuthenticationService.php [hunk 2]
```diff
@@ -3,24 +3,24 @@
...
+use App\Contracts\ParticipantContract;
 use App\ModelRepositoryInterfaces\UserAdminModelRepositoryInterface;
 use App\Contracts\UserGroupContract;
 use Illuminate\Support\Facades\Auth;
 
 
 class AuthenticationService implements AuthenticationContract
 {
-    protected $researchUtility = null;
+    protected $ParticipantUtility = null;
```
Comment (luisjg) [inline app/Services/AuthenticationService.php]: This does not follow `PSR-2` standard

File: app/Services/UserAssignedGroupEmailService.php [hunk 3]
```diff
@@ -7,18 +7,18 @@
...
+use App\Models\Participant;
 
 class UserAssignedGroupEmailService implements UserAssignedGroupEmailContract
 {
     public function sendMail()
     {
         $user = auth()->user();
         $userGroup = UserGroup::find($user['user_id']);
-        $researchID = Research::find($user['user_id']);
+        $ParticipantID = Participant::find($user['user_id']);
```
Comment (luisjg) [inline app/Services/UserAssignedGroupEmailService.php]: This does not follow `PSR-2` standard

File: app/Services/UserCompletesModuleEmailService.php [hunk 4]
```diff
@@ -16,10 +16,10 @@ public function sendMail()
     {
         $user = auth()->user();
         $currentModule = ModuleProgress::find($user['user_id']);
-        $researchID = Research::find($user['user_id']);
+        $ParticipantID = Participant::find($user['user_id']);
```
Comment (luisjg) [inline app/Services/UserCompletesModuleEmailService.php:19]: This does not follow `PSR-2` standard

**Review by luisjg — CHANGES_REQUESTED** [review]
See comments

**Review by luisjg — APPROVED** [review]
👍

**Pull request merged**